cmp/filter/FilterChooserModel.js

```diff
@@ -36,15 +37,18 @@ export class FilterChooserModel extends HoistModel {
     /** @member {Filter[]} */
     @observable.ref favorites = [];
 
-    /** @member {Store} */
-    sourceStore;
+    /** @member {(Store|View)} */
+    valueSource;
```

Comment:
want to make sure I understand why we need to allow binding to the View  -- the View itself could be bound to a store.

Reply:
It was your requirement, Lee :) My understanding is that filtering the cube view may be preferable because it happens before the view calculates its aggregations.